kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -746,38 +839,224 @@ private void addPaneToTray(Pane contentViewPane, Pane slideoutTrayPane) {
     }
 
     public void updateView() {
-        EntityFacade entityFacade = patternViewModel.getPropertyValue(PATTERN);
-        if(entityFacade == null){
-            getStampViewModel().setPropertyValue(MODE, CREATE);
-        }else {
-            getStampViewModel().setPropertyValue(MODE, EDIT);
+        ObjectProperty<EntityFacade> patternProperty = patternViewModel.getProperty(PATTERN);
+        EntityFacade patternFacade = patternProperty.getValue();
+        if (patternFacade == null) {
+            getPatternViewModel().setPropertyValue(MODE, CREATE);
+        } else {
+            ViewCalculator viewCalculator = getViewProperties().calculator();
+            PatternEntity patternEntity = EntityService.get().getEntityFast(patternFacade.nid());
+            getPatternViewModel().setPropertyValue(MODE, EDIT);
+            if (!patternTitleText.textProperty().isBound()) {
+                patternTitleText.textProperty().bind(patternProperty.map(p -> p.description()));
+            }
+            // bind stamp
+            lastUpdatedText.textProperty().bind(getStampViewModel().getProperty(TIME).map(t -> {
+                DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MMM-dd HH:mm:ss");
```

Comment:
This comment is also a question... do we want our code to work on different locales, in other words, is Komet going to be targeted to be used outside of the US as well (each country can have different ways to specify time and date)?

Reply:
I'm not really sure.  @carldea  ?

Reply:
We should really use UTC time, under the covers when writing to the database. But presenting it in local time. We may need to pose the question to the Data team. To be on the safe side, have both presentation and backend calls should be UTC time. But we should create tests to verify that it is consistent.

Reply:
using UTC